.nvmrc

```diff
@@ -1 +1 @@
-14.18.0
+8.12.0
```

Comment:
I guess it's using it to download play-nunjucks, I don't know if the version that play-nunjucks is downloaded under then has any impact on services running play-nunjucks - but v8 is well out of support - will flag if we can have a spike to see if we can safely update the version of node used by play-nunjucks
<img width="747" alt="CleanShot 2022-12-21 at 15 56 03@2x" src="https://user-images.githubusercontent.com/100650/208947867-b320eab6-2a26-4e93-bdb0-ec49c0b3d6b1.png">

Reply:
for now I guess this is ok, because even though we're downgrading versions, actually this is what it was previously being built with in jenkins, is that right @kyle-bowden?